src/encoded/static/components/app.js

```diff
@@ -577,6 +578,10 @@ class App extends React.Component {
         this.setState({ eulaModalVisibility: false });
     }
 
+    closeAccountCreationNotification() {
+        this.setState({ accountCreatedModalVisibility: false });
```

Comment:
I think you might want to intialize 'accountCreatedModalVisibility' the way that you initialize 'accountCreationFailedVisibility' on line 291.

Reply:
Done